kmip/demos/pie/add_attribute.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2015 The Johns Hopkins University/Applied Physics Laboratory
```

Comment:
Given this demo is specifically showing how to add a link attribute, I would rename the script to `add_link_attribute.py`.